tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -246,33 +244,102 @@ def process_resource_set(self, resources, event=None, session=None, token=None):
         return matched
 
     def _get_settings(self, storage_account, session=None, token=None):
-        if self.storage_type == self.BLOB_TYPE:
-            blob_property = get_annotation_prefix(self.BLOB_TYPE)
-            if not (blob_property in storage_account):
-                storage_account[blob_property] = json.loads(jsonpickle.encode(
-                    StorageSettingsUtilities.get_blob_settings(storage_account, token)))
-            return storage_account[blob_property]
-
-        elif self.storage_type == self.FILE_TYPE:
-            file_property = get_annotation_prefix(self.FILE_TYPE)
-            if not (file_property in storage_account):
-                storage_account[file_property] = json.loads(jsonpickle.encode(
-                    StorageSettingsUtilities.get_file_settings(storage_account, session)))
-            return storage_account[file_property]
-
-        elif self.storage_type == self.TABLE_TYPE:
-            table_property = get_annotation_prefix(self.TABLE_TYPE)
-            if not (table_property in storage_account):
-                storage_account[table_property] = json.loads(jsonpickle.encode(
-                    StorageSettingsUtilities.get_table_settings(storage_account, session)))
-            return storage_account[table_property]
-
-        elif self.storage_type == self.QUEUE_TYPE:
-            queue_property = get_annotation_prefix(self.QUEUE_TYPE)
-            if not (queue_property in storage_account):
-                storage_account[queue_property] = json.loads(jsonpickle.encode(
-                    StorageSettingsUtilities.get_queue_settings(storage_account, token)))
-            return storage_account[queue_property]
+        storage_prefix_property = ''
```

Comment:
nit: it can be `storage_prefix_property = get_annotation_prefix(self.storage_type)`, right?

Reply:
Haha true! :D